back/Dockerfile

```diff
@@ -20,35 +20,45 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-      less                                                       \
-      libgeos-dev                                                \
-      libgmp3-dev                                                \
-      libpq-dev                                                  \
-      netcat                                                     \
-      optipng                                                    \
-      pngquant                                                   \
-      shared-mime-info                                           \
- && curl -sL https://deb.nodesource.com/setup_18.x  | bash -     \
- && apt-get -y --no-install-recommends install nodejs            \
+RUN apt-get update                                                             \
+ && apt-get install -qq -y --no-install-recommends                             \
+      build-essential=12.9                                                     \
+      curl=7.74.0-1.3+deb11u7                                                  \
+      # Required to install git dependencies in Gemfile and by bundle-audit
+      git=1:2.30.2-1+deb11u2                                                   \
+      libgeos-dev=3.9.0-1                                                      \
+      libgmp3-dev=2:6.2.1+dfsg-1+deb11u1                                       \
+      # Required by mimemagic on which depends axlsx (3.0.0.pre)
+      shared-mime-info=2.0-1                                                   \
+      # Required by `pg` gem
+      libpq-dev=13.9-0+deb11u1                                                 \
+      # Used to wait for Postgres to be up in CircleCI
+      netcat=1.10-46                                                           \
+      # Image processing
+      imagemagick=8:6.9.11.60+dfsg-1.3+deb11u1                                 \
+      jpegoptim=1.4.6-1                                                        \
+      optipng=0.7.7-1+b1                                                       \
+      pngquant=2.13.1-1                                                        \
+ && curl -sL https://deb.nodesource.com/setup_18.x  | bash -                   \
+ && apt-get -y --no-install-recommends install nodejs=18.14.2-deb-1nodesource1 \
     # Install MJML parser required by email engine.
- && npm install -g mjml@4.4.1                                    \
- && apt-get clean                                                \
+ && npm install -g mjml@4.4.1                                                  \
+ && apt-get clean                                                              \
  && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /cl2_back
 
 RUN gem install bundler:2.3.20
 
 COPY --from=dependency_files /app .
-RUN bundle install
+
+ARG BUNDLER_JOBS
+RUN bundle config set --local clean 'true'               \
+ && bundle config set --local deployment 'true'          \
+ && bundle install ${BUNDLER_JOBS:+--jobs=$BUNDLER_JOBS} \
```

Comment:
It looks like there's already `BUNDLE_JOBS` env var https://bundler.io/v2.4/man/bundle-config.1.html#:~:text=The%20number%20of%20gems%20Bundler%20can

I also thought about giving it a default of 4 (which would be useful for medium resources), but the current default makes sense too.

Reply:
Nice, I didn't know about that one. Will simplify the code.

I'd rather not set a default value to avoid unnecessary coupling between the Dockerfile definition and the executor.

Reply:
Makes sense.